### DISCUSSION TITLE
Too small values display as 0 in Y axis

### DISCUSSION THREAD

**Discussion by nightingale-m** [cetz-package/cetz#262] [Q&A] [answered]
```typst
#let data = (
  (109.9, 0.0084175),
  (129.7, 0.010101),
  (149.5, 0.006734),
  (169.3, 0.0151515),
  (189.1, 0.010101),
)
chart.columnchart(data)
```

![Screenshot_20231020_165112](https://github.com/johannes-wolf/cetz/assets/46788763/11caa21e-1264-4d20-bb22-1d7c474a4d79)

How to make this values display properly?

**Comment by johannes-wolf** [comment, marked as answer]
Sadly, the number of decimals is hard-coded to `1` :( https://github.com/johannes-wolf/cetz/blob/0a9210095e72ec081ef9bc8a707142ef25f6aa65/src/lib/chart.typ#L336.

I will create a patch.

A “workaround” would be using an exponent notation for the ticks:
```typst
  let data = (
    (109.9, 0.0084175),
    (129.7, 0.010101),
    (149.5, 0.006734),
    (169.3, 0.0151515),
    (189.1, 0.010101),
  ).map(r => (r.at(0), r.at(1) * calc.pow(10, 4)))


  columnchart(size: (10, 10),
    y-unit: $times 10^(-4)$,
    data)
```

![grafik](https://github.com/johannes-wolf/cetz/assets/519002/43243568-c275-401c-b183-4db1615211f2)

**Reply by nightingale-m** [reply]
Thanks for the help!
